core/field_angle.ts

```diff
@@ -278,9 +278,9 @@ export class FieldAngle extends FieldInput<number> {
     // a click handler on the drag surface to update the value if the surface
     // is clicked.
     this.clickSurfaceWrapper_ = browserEvents.conditionalBind(
-        circle, 'click', this, this.onMouseMove_, true, true);
+        circle, 'pointerdown', this, this.onMouseMove_, true, true);
```

Comment:
Does this preserve the behaviour?

Reply:
Same question for all `click` -> `pointerdown` conversions.

Reply:
Yes, I manually verified that all cases where this swap was made appear to be working correctly.